test/uk/gov/hmrc/nisp/events/QuestionnaireEventSpec.scala

```diff
@@ -26,17 +26,22 @@ class QuestionnaireEventSpec extends UnitSpec {
    "QuestionnaireEvent" should {
 
      "not store the email address if research is No" in {
-       val event = new QuestionnaireEvent(None, None, None, None, None, None, Some(1), Some("test@test.com"), "Name", "NINO", "")
+       val event = new QuestionnaireEvent(None, None, None, None, None, None, None, Some("Lorem ipsum"), Some(1), Some("test@test.com"), "Name", "NINO", "")
        event.detail("email") shouldBe "N/A"
      }
 
      "not store the email address if research is Blank" in {
-       val event = new QuestionnaireEvent(None, None, None, None, None, None, None, Some("test@test.com"), "Name", "NINO", "")
+       val event = new QuestionnaireEvent(None, None, None, None, None, None, None, Some("Lorem ipsum"), None, Some("test@test.com"), "Name", "NINO", "")
```

Comment:
But that's not what this test is testing.
They also haven't got `Some(8)` in `whatWillYouDoNext` so it's not even a realistic event.